src/console_node/httpUtils.go

```diff
@@ -121,7 +121,7 @@ func getURL(URL string, requestHeaders map[string]string) ([]byte, int, error) {
 		log.Printf("getURL Error on request to %s: %s", URL, err)
 		return nil, -1, err
 	}
-	log.Printf("getURL Response Status code: %d\n", resp.StatusCode)
+	//log.Printf("getURL Response Status code: %d\n", resp.StatusCode)
```

Comment:
Same question -- should we just remove this? Or put it behind some conditional, if you're keeping it here for future debugging?

Reply:
Ibid.